### ISSUE TITLE
PD Runner not working with PD 18.2.0

### ISSUE THREAD

**Issue by KohlerVG** [utsanjan/PD-Runner#72]
PD Runner doesn't work with the latest PD 18.2.0.

**Comment by matantczak** [comment]
Is it possible to install backward version of PD?

**Comment by EaterComputer** [comment]
> Ok I solved it. Just instaled once again PD 18.1.1 from link: https://download.parallels.com/desktop/v18/18.1.1-53328/ParallelsDesktop-18.1.1-53328.dmg Then once again reinstalled PD-Runner

Yup. That works. When you receive a "new update" windows when installing, close, then click keep current, and it will downgrade to 18.1.1

**Comment by inzar98** [comment]
but thats temp solution ofc. We are waiting for new update ^^ Thx for your efforts guys
